src/lib.rs

```diff
@@ -134,4 +204,18 @@ mod tests {
             v == d
         }
     }
+
```

Comment:
Thank you for writing the test code!

The test functions also need `#[cfg(feature = "bindgen")]`.

Additionally, it fails the code format check, so that needs to be fixed as well.
```suggestion

#[cfg(feature = "bindgen")]
```